decentraland/unity-explorer
# fix: Avoid texture format crash

## What does this PR change?

Fixes #510

Some wearables (as described in bug) come with differente texture format than expected. As a contigency, if this is the case, we set the default texture.

Ideally the system would be robust to handle this scenarios, a refactor has been described as [issue ](https://github.com/decentraland/unity-explorer/issues/816) to be done if needed.


## How to test the changes?

TEST IN MAC AND WINDOWS IN EDITOR

1. Launch the explorer 
2. Instantiate an avatar with each of this urn using the Avatar Creator in Debug menu 
urn:decentraland:matic:collections-v2:0x36e94d12899317dc792e558f993213eef9b85c78:0:68 => https://decentraland.org/marketplace/contracts/0x36e94d12899317dc792e558f993213eef9b85c78/items/0
urn:decentraland:matic:collections-v2:0x3786b880f988b04c8f7a88443b25c777bcfb325e:0 => https://decentraland.org/marketplace/contracts/0x3786b880f988b04c8f7a88443b25c777bcfb325e/tokens/34
3. Check that you see the mask and boots respectively with no editor crash

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
